cast/tests/e2e/main.rs

```diff
@@ -0,0 +1,2 @@
+mod declare;
+pub(crate) mod helpers;
```

Comment:
I think `crate` is unnecessary

Reply:
Hmm ok. I remember that I added it in one of my previous PRs and got a comment that it is unnecessary. I think it specifies the visibility so we can keep it if you want